src/preamble.js

```diff
@@ -267,15 +267,34 @@ var HEAP,
   HEAPF64;
 
 function updateGlobalBufferAndViews(buf) {
+  // Using Data Views to force little-endian ordering on all architectures.
+  function _generateProxyObj(byteCount, getter, setter) {
+    return {
+      get: function (target, key) {
+        return new DataView(target.buffer)[getter](byteCount * (key), true);
+      },
+      set: function (target, key, value) {
+        new DataView(target.buffer)[setter](byteCount * (key), value, true);
```

Comment:
Hmm, isn't it super-inefficient to allocate a new dataview on every single memory load and store operation?